tests/scripts/components-configuration-crypto.sh

```diff
@@ -245,11 +245,11 @@ component_test_psa_external_rng_no_drbg_use_psa () {
     scripts/config.py unset MBEDTLS_CTR_DRBG_C
     scripts/config.py unset MBEDTLS_HMAC_DRBG_C
     scripts/config.py unset PSA_WANT_ALG_DETERMINISTIC_ECDSA # Requires HMAC_DRBG
-    CC=$ASAN_CC cmake -D CMAKE_BUILD_TYPE:String=Asan .
+    cmake -D CMAKE_BUILD_TYPE:String=Asan .
```

Comment:
CMAKE_BUILD_TYPE does not imply the compiler, the value of $ASAN_CC is dictated by all.sh, we should keep it.